source/views/student-orgs/list.js

```diff
@@ -192,7 +192,7 @@ export class StudentOrgsView extends React.Component {
   }
 
   render() {
-    if (this.state.loading) {
+    if (this.state.loading && !this.state.refreshing) {
```

Comment:
Instead of this, you'll want to remove these lines rom fetchData:

https://github.com/StoDevX/AAO-React-Native/blob/b8531c8c710e9ed5d44e159fbcec1ce747a3af87/source/views/student-orgs/list.js#L98

I can do this in a bit, since I think you're indisposed tonight.

Reply:
That would be appreciated, thanks!